assets/i18n/locales/zh-hk.json

```diff
@@ -15,6 +17,15 @@
   "invites/error_title": "邀請錯誤",
   "invites/error": "您的訪問出現問題，請重試或聯絡{{email}}",
   "invites/placeholder": "邀請",
+  "locationData/failed_upload: 上載衰": "{{value}}",
```

Comment:
You might want to check the quotation placement on some of these terms. I think they are off slightly.
I have noticed this in the other PR's as well. Thanks for adding these!